cucumber/api/features/secrets.feature

```diff
@@ -17,7 +17,15 @@ Feature: Adding and fetching secrets
     Given I am a user named "eve"
     Given I create a new "variable" resource called "probe"
 
-  Scenario: Fetching a resource with no secret values returna a 404 error.
+  Scenario: Update a secret for a resource with no permissions
+    When I am a user named "alice"
+    When I POST "/secrets/cucumber/variable/probe" with body:
+    """
+    v-1
+    """
+    Then the HTTP response status code is 404
+
+  Scenario: Fetching a resource with no secret values return a 404 error.
 
```

Comment:
please remove this empty line

Reply:
I also prefer without the empty line however I prefer to keep the style of the file the same for all scenarios and we can create a separate PR to fix the tests style.

Reply:
i think that this is a small change that can go in this PR but am leaving it up to you